include/vrv/toolkit.h

```diff
@@ -570,8 +570,11 @@ class Toolkit {
      *
      * This can be called once the rendering option were changed, for example with a new page (sceen) height or a new
      * zoom level.
+     *
+     * @param jsonOptions A stringified JSON object with the action options
+     * resetCache: true or false; true by default;
```

Comment:
Does this mean that the argument is a string with the value of `"true"` or `"false"` (instead of actual boolean options `true` or `false`)?

Reply:
No, it is only the entire JSON object that is a string, not the values. In C++, you will do
```cpp
toolkit.RedoLayout("{ 'resetCache': true }");
```
In JS
```js
toolkit.redoLayout({ 'resetCache': true });
```
That is how it is for all options passed as JS. That is, they are a string in the C++ codebase, but a JSON object in the JS interface.
Can you propose a better wording?

Reply:
(basically  the same here https://github.com/rism-digital/verovio/pull/2606)